### ISSUE TITLE
`weave stop` can leave router running

### ISSUE THREAD

**Issue by bboreham** [weaveworks/weave#1576]
If $DOCKER_HOST is pointing at the proxy, `weave stop` will stop the proxy, which will cause all further docker commands to fail, so the router is left running.

If we stop them the other way round, this will give less surprising results.

**Comment by bboreham** [comment]
Closed via 2e54efe34d3be15c7c6bfe7802e1857d2a79c574

**Comment by rade** [comment]
> If $DOCKER_HOST is pointing at the proxy, weave stop will stop the proxy, which will cause all further docker commands to fail, so the router is left running.

That is not true. The script is talking to docker via the socket, not the proxy.

```
$ weave launch
$ eval $(weave env)
$ weave stop
WARNING: It appears that your environment is configured to use the Weave Docker API proxy. Stopping it will break this and subsequent docker invocations. To restore your environment, run 'eval $(weave env --restore)'.
Error deleting container: Cannot connect to the Docker daemon. Is 'docker -d' running on this host?
Cannot connect to the Docker daemon. Is 'docker -d' running on this host?
$ eval $(weave env --restore)
$ docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
```
